### PR TITLE
Settlement borrowing module

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

File: guides/app-guides/app-faq.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ In the case there is not enough funds in a pool Hedging Agents are withdrawing f
 
 All collateral/agTokens pools are separated and managed by independent contracts. This means that, even though pools for a given collateral can use the same strategies, it could be the case that they don't or that some parameters related to these strategies differ.
 
-Additionally, the amounts put by users minting on these pools differ, which impact the multiplier effect explained [here](https://docs.angle.money/concepts/standard-liquidity-providers#multiplier-effect), and change the APYs for SLPs.
+Additionally, the amounts put by users minting on these pools differ, which impact the multiplier effect explained [here](/core-module/standard-liquidity-providers/README.md#✖-multiplier-effect), and change the APYs for SLPs.
```
Comment (sogipec) [inline guides/app-guides/app-faq.md:52]: Link will be broken here

File: borrowing-module/vaults/settlement.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+description: Settlement - Stopping all operations for a specific vault type
+---
+
+# Vault Settlement
+
+As in the Core module, the Borrowing module has a settlement mechanism to stop unsustainable vault types and pay back users. Its goal is to ensure that agTokens issued through this module are **always backed by collateral**. 
+
+## Idea
+The vault settlement should be used only in extreme cases, for example when vaults from a specific [`VaultManager`](/borrowing-module/glossary.md) contract start to be under-collateralized and liquidator stop liquidating unhealthy positions.
```
Comment (sogipec) [inline borrowing-module/vaults/settlement.md:10]: Need to link to Maker global settlement and to the fact that by liquidating all vault contracts at the same time, you can technically put an end to a stablecoin

Comment (sogipec) [reply borrowing-module/vaults/settlement.md:10]: Also need to insist that it's the place where stablecoins can be redeemed at oracle value, meaning that it's part of what makes agTokens ultimately stable

Comment (sogipec) [reply borrowing-module/vaults/settlement.md:10]: Some "éléments de langage" can be taken from here: https://blog.makerdao.com/introduction-to-emergency-shutdown-in-multi-collateral-dai/

Comment (tutacrypto) [reply borrowing-module/vaults/settlement.md:10]: For the 1st point, it's not the case for us as there is the other module
We would have to do settlements in both modules at the same time for that

File: guides/app-guides/app-faq.md (hunk 1 above)
Comment (sogipec) [reply guides/app-guides/app-faq.md:52]: Yes but core-module is not on the same repo: you need to do something like ../../core-module

File: borrowing-module/vaults/settlement.md (hunk 2 above)
Comment (sogipec) [reply borrowing-module/vaults/settlement.md:10]: Yes so to the module

**Pull request merged**